go/test/endtoend/vtgate/queries/aggregation/aggregation_test.go

```diff
@@ -18,6 +18,7 @@ package aggregation
 
 import (
 	"fmt"
+	"math/rand"
```

Comment:
We should use rand/v2 in general everywhere. I moved everything to that not too long ago already.

Reply:
https://github.com/vitessio/vitess/pull/16264 to actually lint for this :smile:.